marketplace_builder/migrations/20181213205000_migrate_contributors.liquid

```diff
@@ -46,7 +46,7 @@
   },
   {
     "slug": "get-started/setup/development-workflow",
-    "new_slug": "how-platformos-works/development-workflow"
+    "new_slug": "how-platformos-works/platformos-workflow/development-workflow"
```

Comment:
I dont think this will work, meaning, this migration is marked as done and wont be run again.
@lemingos - correct me if im wrong

Reply:
you are correct @pavelloz - need to do new migration

Reply:
@lemingos Shall I create the new migration after this PR has been merged?